lib/MPIPreferences/src/MPIPreferences.jl

```diff
@@ -149,8 +149,8 @@ function identify_abi(libmpi)
         if (m = match(r"^FUJITSU MPI Library (\d+.\d+.\d+)", version_string)) !== nothing
             version = VersionNumber(m.captures[1])
         end
-    elseif startswith(version_string, "MPIwrapper")
-        impl = "MPIwrapper"
+    elseif startswith(version_string, "MPItrampoline")
```

Comment:
Is this change necessary? I think this is the only one I didn't have in my PR

Reply:
The `version_string` remains `MPIwrapper`.